api/src/pcapi/routes/serialization/finance_serialize.py

```diff
@@ -53,3 +60,65 @@ def from_orm(cls, invoice: finance_models.Invoice) -> "InvoiceResponseModel":
 
 class InvoiceListResponseModel(BaseModel):
     __root__: list[InvoiceResponseModel]
+
+
+class LinkedVenues(BaseModel):
+    """A venue that is already linked to a bank account."""
+
+    id: int
+    common_name: str
+
+    class Config:
+        orm_mode = True
+
+
+class ManagedVenues(BaseModel):
+    id: int
+    common_name: str
+    siret: str
+    bankAccountId: int | None
+
+    class Config:
+        orm_mode = True
+
+    @classmethod
+    def from_orm(cls, venue: "offerers_models.Venue") -> "ManagedVenues":
+        """
+        For each managed venue, we only want to display the actual bank account
+        that is linked to it, if any.
+        """
+        if len(venue.bankAccountLinks) > 1:
+            raise AssertionError("There should be one or none bank account.")
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**🎯 suggestion:** ‏Je crois que tu ne test pas ça
On veut remonter quoi au front dans ce cas ? On risque pas de prendre une 500 ici ?

Reply:
La requête s’assure de ça (`sqla.func.upper(models.VenueBankAccountLink.timespan).is_(None)`) et si c’est enlevé par erreur ça cassera dans les tests puisqu’on lèvera cette exception au moment de la serialization @lixxday

Reply:
Vu ensemble, je change pour `logger.error` pour éviter que l’utilisateur perde l’accès à son espace tout en permettant que l’on soit au courant si ce corner case devrait se produire